src/game/scenes/mechlab/lab_menu_main.c

```diff
@@ -252,7 +266,7 @@ static const spritebutton_tick_cb tick_cbs[] = {
     lab_menu_tick_in_tournament, // lab_menu_tick_delete,
     lab_menu_tick_in_tournament, // lab_menu_tick_sim,
     NULL,                        // lab_menu_tick_quit,
-    lab_menu_tick_in_tournament, // lab_menu_tick_tournament,
```

Comment:
This should be disabled when there's not a player loaded, at least.

Reply:
The loop in lab_menu_main_create handles it.

that loop now also now sets supports_select, so the buttons the loop disables can't be selected.